src/test/unit/string.test.js

```diff
@@ -144,4 +144,48 @@ describe('utils/string', function() {
       expect(removeURLs(string, false)).toEqual(string);
     });
   });
+
+  describe('removeFilePath', function() {
+    it('should remove Unix-like paths', () => {
+      // A file in the root dir.
+      let string = '/test.txt';
+      expect(removeFilePath(string)).toEqual('<PATH>/test.txt');
+      // A file in a non-root dir.
+      string = '/var/test.txt';
+      expect(removeFilePath(string)).toEqual('<PATH>/test.txt');
+      // A file in the Linux home dir.
+      string = 'home/username/test.txt';
+      expect(removeFilePath(string)).toEqual('<PATH>/test.txt');
+      // A file in a Unix derived home dir.
+      string = 'users/username/test.txt';
+      expect(removeFilePath(string)).toEqual('<PATH>/test.txt');
+      // A file in another Unix derived home dir.
+      string = 'user/username/test.txt';
+      expect(removeFilePath(string)).toEqual('<PATH>/test.txt');
+      // A file in the macOS home dir.
+      string = '/Users/username/test.txt';
```

Comment:
I'm not sure what all these testcases bring? (from `users/username/test.txt`). They look redundant with the previous tests, right?